pyproject.toml

```
@@ -39,7 +39,7 @@ test = ["pytest>=6.0", "pytest-cov", "pytest-qt"]
 pyqt5 = ["PyQt5"]
 pyside2 = ["PySide2"]
 pyqt6 = ["PyQt6"]
-pyside6 = ["PySide6!=6.4.0,!=6.4.0.1"]
+pyside6 = ["PySide6!=6.4.0,!=6.4.0.1,!=6.4.1"]
```

```suggestion
pyside6 = ["PySide6"]
```

(and then apply fix https://github.com/pymmcore-plus/pymmcore-widgets/pull/71/files#r1030902381)